### ISSUE TITLE
Add a visual indictor for non-target shape detection

### ISSUE THREAD

**Issue by BLFiedler** [phetsims/quadrilateral#122]
To bring visual in parity with the auditory display, we'll need a visual indicator for when a non-target shape is detected. Currently, an auditory tick plays for non-parallelogram shapes.

It should be appropriately subtle and much less interesting than the green color change of the parallelogram.

I was thinking of 2 options.

1. Since we're using color feedback for the parallelogram, let's continue that trend and add a much subtler color change "flash" in the same family as the default non-parallelogram yellow. We currently use RGB 255,217,102. What if we desaturated to RGB 255,239,189. The flash of color should last briefly, on the order of the duration of the sound that plays.
![image](https://user-images.githubusercontent.com/47331395/162016909-f3fd592c-cd17-4a03-9cd7-5f545ef0ebe0.png)

2. Animate the shape detection with a subtle, transient (~500ms or less) narrowing of the side width and vertex diameter by say 5%. A sort of "lock" effect.

Any feelings on what might work out better in this instance @terracoda?

**Comment by jessegreenberg** [comment]
In https://github.com/phetsims/quadrilateral/commit/d24a85779aece28dc2f590e688a21ff41a6aa003 I made it so the shape is the desaturated color when it is not a parallelogram and is a named shape. In https://github.com/phetsims/quadrilateral/commit/87bb5820061c06cf61362a7a68e0df922447e971 I tried changing it so that the desaturated color is only shown for 0.35 seconds.

Over to @BLFiedler to try it out or keep considering.

**Comment by BLFiedler** [comment]
This will work for now until we get more robust feedback from codesign studies on how we might provide feedback on the hierarchy (toward separate cases or toward inclusive property feedback).

Secondarily, we would like the ability to hide this feedback along with the sound for interviews/codesign. This will happen in issue #129
